devices/wholeBodyDynamics/README.md

```diff
@@ -14,7 +14,7 @@ For an overview on `wholeBodyDynamics` and to understand how to run the device,
 |:--------------:|:--------------:|:-----------------:|:-----:|:-------------:|:--------:|:-----------------------------------------------------------------:|:-----:|
 | axesNames      |      -         | vector of strings |   -   |   -           | Yes      | Ordered list of the axes that are part of the remapped device.    |       |
 | modelFile      |      -         | path to file      |   -   | model.urdf    | No       | Path to the URDF file used for the kinematic and dynamic model.   |       |
-| assume_fixed    |                | frame name        |   -   |     -         | No       | If it is present, the initial kinematic source used for estimation will be that specified frame is fixed, and its gravity is specified by fixedFrameGravity. Otherwise, the default IMU will be used. | |
+| assume_fixed    |                | frame name        |   -   |     -         | No       | If it is present, the initial kinematic source used for estimation will be that specified frame is fixed, and its gravity is specified by fixedFrameGravity. The IMU related parameters are overriden if this parameter exists. Otherwise, the default IMU will be used. | |
```

Comment:
Are overriden in which sense? With which values are overriden?

Reply:
Overridden in the sense, those settings from the configuration file related to the IMU would never be used.

Reply:
Fixed in https://github.com/robotology/whole-body-estimators/pull/95/commits/f6c3a8122a6a3aaa4582f2faf8c5ec13bf54c5a2